<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddIGMASStationDialog</class>
 <widget class="QDialog" name="AddIGMASStationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="infoContainer">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>230</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>iGMAS测站信息</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>MOUNTPOINT：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="stationMountEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>IP(DEC)：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="stationIPEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Port(DEC)：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="stationPortEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>6</number>
    </property>
    <item>
     <widget class="QPushButton" name="confirmButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>230</width>
     <height>130</height>
    </rect>
   </property>
   <property name="title">
    <string>iGMAS测站位置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>经度(度)：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="stationLongitudeEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>纬度(度)：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="stationLatitudeEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>高度(米)：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="stationHeightEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>230</width>
     <height>50</height>
    </rect>
   </property>
   <property name="title">
    <string>iGMAS测站配置</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>可用性：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="stationAvailableBox">
      <item>
       <property name="text">
        <string>不可用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>可用</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>470</width>
     <height>60</height>
    </rect>
   </property>
   <property name="title">
    <string>iGMAS测站登录配置</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,0,0">
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>用户名：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stationUserNameEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>密码：</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="stationPasswordEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>stationMountEdit</tabstop>
  <tabstop>stationIPEdit</tabstop>
  <tabstop>stationPortEdit</tabstop>
  <tabstop>stationLongitudeEdit</tabstop>
  <tabstop>stationLatitudeEdit</tabstop>
  <tabstop>stationHeightEdit</tabstop>
  <tabstop>stationUserNameEdit</tabstop>
  <tabstop>stationPasswordEdit</tabstop>
  <tabstop>confirmButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>stationAvailableBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
